build/helper/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM debian:stable-slim
```

Comment:
Please specify the Debian release; `buster-slim`; but thanks for using Debian!

Reply:
changed